### PR TITLE
HTTP and MQ service for GET hooked up

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

File: mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Creates and configures a message service listener for the data retrieval service
+ * 
+ * @author dmsibley
```
Comment (mhines-usgs) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java]: lol

Comment (davidmsibley) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java]: I will always live on in copy and pastes.

File: mlr-service/src/main/java/gov/usgs/wma/messaging/MicroserviceUtils.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+/**
+ * Utility class that wraps and aids with basic microservice architecture
+ * management and configuration.
+ * 
+ * @author thongsav
+ */
+public class MicroserviceUtils {
+	private static final Logger log = LoggerFactory.getLogger(MicroserviceUtils.class);
+	
+	private static final String DATA_QUEUE_TIME_JNDI_NAME = "aqcu.mailbox.ttl.ms";
```
Comment (isuftin) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MicroserviceUtils.java]: This is not the AQCU project

File: mlr-service/src/main/java/gov/usgs/wma/util/MlrGsonFactory.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * A factory class for the construction of properly configured Gson objects for
+ * JSON serialization and deserialization.
+ * 
+ * @author thongsav
+ */
+public class MlrGsonFactory {
+
+	/**
+	 * Returns a Gson object that is properly configured for use in AQCU.
```
Comment (isuftin) [inline mlr-service/src/main/java/gov/usgs/wma/util/MlrGsonFactory.java]: This is not the AQCU project

File: mlr-service/src/main/java/gov/usgs/wma/util/MlrGsonFactory.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	 * 
+	 * @return
+	 */
+	public static Gson getConfiguredGson() {
+		return new GsonBuilder()
+				//TODO may need Time adapters, see AQCU's GSON Factory if so
```
Comment (isuftin) [inline mlr-service/src/main/java/gov/usgs/wma/util/MlrGsonFactory.java]: This is not the AQCU project

File: mlr-service/src/main/java/gov/usgs/wma/webservice/MlrWebservicesEntryPoint.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+		// features
+
+		// Add additional features such as support for Multipart.
+		classes.add(MultiPartFeature.class);
+
+		// error handling providers
+		classes.add(GenericExceptionMapper.class);
+		
+		
+		log.trace("AqcuFrontEndWebservicesEntryPoint application started");
```
Comment (isuftin) [inline mlr-service/src/main/java/gov/usgs/wma/webservice/MlrWebservicesEntryPoint.java]: This is not the AQCU project

File: mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java (hunk 1 above)
Comment (thongsav-usgs) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java]: copy pasta!

File: mlr-service/src/main/java/gov/usgs/wma/messaging/MicroserviceUtils.java (hunk 2 above)
Comment (thongsav-usgs) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MicroserviceUtils.java]: good catch

File: mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java (hunk 1 above)
Comment (thongsav-usgs) [inline mlr-service/src/main/java/gov/usgs/wma/messaging/MsgserviceListener.java]: You will live on forever.

**Pull request merged**